src/chromium/interfaces/contextMenus.js

```diff
@@ -105,9 +105,9 @@ export async function handleChangeDefaultLaunchContextMenuClick() {
  * Handles the context menu clicks for the Chromium Extension.
  *
  * @param {*} info  The context menu item info object.
- * @param {*} tab  The tab object to launch the browser with.
+ * @param {string} url  The url the context menu will handle.
  */
-export async function handlePlatformContextMenuClick(info, tab) {
+export async function handlePlatformContextMenuClick(info, url) {
```

Comment:
This method sounds like the click handler of the context menu, in which case `tab` would be the correct signature.

info.linkUrl is only meaningful for context menus for the "link" target. For clarity, you could consider only dereferencing info.linkUrl for link menus.